src/modules/graphql/resolvers/class/services/default-class-schedules.ts

```diff
@@ -147,7 +147,7 @@ const indx22: TimeSlotDbModel = {
 const indx23: TimeSlotDbModel = {
   index: '8_5',
   hours: '12:45 - 13:15',
-  lesson: lunchLesson,
+  lesson: dismissLesson,
```

Comment:
Where do you refer to "dismissLesson"?

Reply:
@eran1205 
this is a constant on the default-schedule-lessons.ts that refer to פיזור תלמידים
the bug requested to replace lunch with dismissLesson on that schedule slot.

I have updated the dbSeed.,